runWPT.sh

```diff
@@ -50,18 +61,37 @@ else
   log "Running WPT in headful mode..."
 fi
 
-# Go to the project root folder.
-(cd "$(dirname "$0")/" && \
-./wpt/wpt run \
-  --binary "$WPT_BROWSER_PATH" \
-  --webdriver-binary runBiDiServer.sh \
-  --webdriver-arg="--headless=$HEADLESS" \
-  --log-wptreport "$WPT_REPORT" \
-  --manifest "$MANIFEST" \
-  --metadata "$WPT_METADATA" \
-  --timeout-multiplier 8 \
-  "$PRODUCT" \
-  "$@")
+declare -a WPT_RUN_ARGS=(
+  --binary "$CHROME_BIN"
+  --webdriver-binary runBiDiServer.sh
+  --webdriver-arg="--headless=$HEADLESS"
+  --log-wptreport "$WPT_REPORT"
+  --manifest "$MANIFEST"
+  --metadata "$WPT_METADATA"
+  --timeout-multiplier "$TIMEOUT_MULTIPLIER"
+)
+
+if [[ "$CHROMEDRIVER" == "true" ]]; then
+  log "Using Chromedriver with mapper..."
+  WPT_RUN_ARGS+=(
```

Comment:
I could not make this work inline together with `set -u`. Making it a separate variable was easier.